source/android/mobile/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,7 @@
             android:configChanges="orientation|screenSize|keyboardHidden">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
+                <action android:name="android.media.action.MEDIA_PLAY_FROM_SEARCH" />
```

Comment:
What is this for? I thought this was for apps that want to play music.

Reply:
"To support voice searches on Android Auto, you should also register an intent-filter for the action android.media.action.MEDIA_PLAY_FROM_SEARCH."

---
In reply to: [135900329](https://github.com/Microsoft/AdaptiveCards/pull/588#discussion_r135900329) [](ancestors = 135900329)